test/utils.ts

```diff
@@ -0,0 +1,80 @@
+/*
+ * Copyright OpenSearch Contributors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+export const mockStore = {
+  getState: () => ({
+    opensearch: {
+      errorMessage: '',
+    },
+    workflows: {
+      loading: false,
+      errorMessage: '',
+      workflows: {
+        '12345': {
```

Comment:
Suggest to move this entire workflow JSON into a constant. Or better yet, a util fn to generate a dummy workflow based on limited input. For example,
```
function generateWorkflow(workflowId: string, workflowName: string): Workflow
```

Reply:
Eventually these types of functions can get more complex, take in more parameters, etc. For example, workflows with multiple processors, ingest disabled, different index settings, etc.